services/core-api/app/config.py

```diff
@@ -157,6 +159,13 @@ def JWT_ROLE_CALLBACK(jwt_dict):
     EMAIL_ENABLED = os.environ.get('EMAIL_ENABLED', False)
     EMAIL_RECIPIENT_OVERRIDE = os.environ.get('EMAIL_RECIPIENT_OVERRIDE')
 
+    #Templates
+    TEMPLATE_FOLDER_BASE = os.environ.get('TEMPLATE_FOLDER_BASE', '/app/app/templates')
+
+    TEMPLATE_FOLDER_IRT = os.environ.get('TEMPLATE_FOLDER_IRT', f'{TEMPLATE_FOLDER_BASE}/projects/')
```

Comment:
I notice we have both "project" and "projects" folders in the template folder. Can we clean one of those up that we aren't using?

Reply:
Having just projects folder.